src/main/java/net/minecraftforge/client/event/ClientChatReceivedEvent.java

```diff
@@ -66,8 +65,7 @@ public ChatType getType()
      * The UUID of the player or entity that sent this message, or null if not known.
      * This will be equal to {@link net.minecraft.util.Util#field_240973_b_} for system messages.
      */
-    @Nullable
-    public UUID getSenderUUID()
+    public UUID getUUID()
```

Comment:
What is the reason behind this name change? `getUUID` is fairly unspecific...

Reply:
Yeah, on hindsight, the original method is better named. Will fix.